scripts/build/generateIcons.ts

```diff
@@ -201,9 +201,11 @@ async function loadCodiconMappings(): Promise<Record<string, number | undefined>
     const codicons = path.join(projectDir, 'node_modules', '@vscode', 'codicons', 'src')
     const data = JSON.parse(await fs.readFile(path.join(codicons, 'template', 'mapping.json'), 'utf-8'))
     const mappings: Record<string, number | undefined> = {}
+    let counter = 0xf000
```

Comment:
I think we should change line 105 to `0xf000` instead. That keeps the codicon mappings 1:1. It'll regenerate `package.json` but that's OK.